plugins/manager/manager.go

```diff
@@ -176,7 +176,7 @@ func (pm *PluginManager) dispensePlugins() error {
 // launchInternalPlugin is used to dispense internal plugins.
 func (pm *PluginManager) launchInternalPlugin(id plugins.PluginID, info *pluginInfo) (PluginInstance, *base.PluginInfo, error) {
 
-	raw := info.factory(pm.logger.ResetNamed("internal_plugin"))
+	raw := info.factory(pm.logger.ResetNamed("internal_plugin" + id.Name))
```

Comment:
Adding a separator here, otherwise it looks like this:

```
2020-04-24T11:21:15.875-0400 [INFO]  internal_pluginprometheus: hi
```

```suggestion
	raw := info.factory(pm.logger.ResetNamed("internal_plugin." + id.Name))
```